framer/Material-UI.framerfx/code/ThemeProvider.tsx

```diff
@@ -1,31 +1,20 @@
 import * as React from 'react';
 import { addPropertyControls, ControlType } from 'framer';
-// tslint:disable-next-line: ban-ts-ignore
-// @ts-ignore
 import { MuiThemeProvider, createMuiTheme } from '@material-ui/core/styles';
-import { parseColor } from './utils/parseColor';
+import { parseColor } from './utils';
```

Comment:
Having it in a subfolder kept it separate from the code components in the Framer UI.

Reply:
god framer, why :laughing: 

Will put it in `util/index` I guess. No need to create a file per util.

Reply:
done